data/wanderer/wanderers start.txt

```diff
@@ -224,10 +224,12 @@ mission "Wanderers: Diplomacy"
 
 
 mission "Wanderers: Translation Machine"
-	description "Sayari, the Hai ambassador, suggested that you visit <destination>, where Hai technologists are working on translation machines."
+	description "Find language training data at <destination> so the Wanderers can create an automated language translator."
 	landing
 	source "Vara K'chrai"
-	destination "Greenwater"
+	destination "Alexandria"
+	substitutions
+		"<Library search>" `Travelling through the library in search for data suitable for training the Wanderer translation algorithm raises an unexpected problem: you're surrounded by it. You guessed that even fifty Bulk Freighters wouldn't be enough to carry all the tomes and data cores from the Alexandria library. Asking around, you're referred to a research librarian who develops automated translation algorithms for dead languages.`
```

Comment:
```suggestion
		"<Library search>" `Traveling through the library in search for data suitable for training the Wanderer translation algorithm raises an unexpected problem: you're surrounded by it. You guess that even fifty Bulk Freighters wouldn't be enough to carry all the tomes and data cores from the Alexandria library. Asking around, you're referred to a research librarian who develops automated translation algorithms for dead languages.`
```

Reply:
I already fixed the Travelling spelling error, but I wanted to highlight the "guess" vs "guessed" change I'd like to make.